src/main/java/sirius/biz/storage/layer2/BlobDispatcher.java

```diff
@@ -183,34 +188,33 @@ private void physicalDelivery(WebContext request, BlobUri blobUri) {
     }
 
     /**
-     * Checks if the provided accessToken is invalid and calculate the cache headers.
+     * Checks if the provided accessToken is invalid and return the cache time in seconds based on the hash validity.
      *
      * @param response    the response to return
      * @param accessToken the security token to verify
      * @param key         the key to verify
      * @param space       the space which is accessed
-     * @return <tt>true</tt> if the accessToken is invalid, <tt>false</tt> otherwise
+     * @return the cache time in seconds or <tt>null</tt> if the hash is invalid
      */
-    private boolean checkHashInvalid(Response response, String accessToken, String key, String space) {
+    private Integer checkHashInvalid(Response response, String accessToken, String key, String space) {
```

Comment:
Maybe, it would be good to adapt the name of the function, considering the new returned payload.